src/carousel.tsx

```diff
@@ -347,9 +350,18 @@ const BaseCarousel: React.FC<BaseCarouselProps> = ({
                     shownItemIndexes.unshift(idx);
                 }
             }
-            onPageChange({pageIndex: currentPageIndex, shownItemIndexes});
+
+            if (!pageInitialized.current) {
+                pageInitialized.current =
+                    lastPageIndex.current !== currentPageIndex &&
+                    shownItemIndexes.includes(initialActiveItem || 0);
```

Comment:
This condition changed due to a bug causing a test to fail: the carousel should be considered initialized if a page transition ends with the initial item being contained on the current page.